NewForce-at-Mountwest/bangazon-api-farfalle
# Nm controllers

# Description

Created GET all orders, GET single order, PUT, POST, and DELETE methods for orders.
Created additions for include=customers, include=products, completed=true, completed=false.

## Related Ticket(s)

Ticket 4
https://github.com/NewForce-at-Mountwest/bangazon-api-farfalle/issues/7

## Steps to Test

1- Press Ctrl+f5
2- Go to https://localhost:5001/api/Orders in the browser
3- You should see all Orders
4- Go to https://localhost:5001/api/Orders/1 in the browser
5- You should see a single order
6- Go to https://localhost:5001/api/Orders?include=customers in the browser
7- You should see all orders with the customers expanded
8- Go to https://localhost:5001/api/Orders?include=products in the browser
9- You should see all orders with products expanded
10- Open Postman
11- Post to https://localhost:5001/api/Orders with the header:    
{ 
"customerId": 3,
"paymentTypeId": 3
}
12-You should see a new order at https://localhost:5001/api/Orders
13- Put to https://localhost:5001/api/Orders/1 with the header:
{ 
"customerId": 1,
"paymentTypeId": 1
}
14-You should see that order 1 at https://localhost:5001/api/Orders/1 has changed
15- Delete to https://localhost:5001/api/Orders/2
16- Order number 2 should be deleted

# Checklist:
- [N/A] I have made corresponding changes to the documentation
